proto/dolt/services/eventsapi/v1alpha1/client_event.proto

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 Liquidata, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package dolt.services.eventsapi.v1alpha1;
+
+option go_package = "dolt/services/eventsapi_v1alpha1;eventsapi";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+service ClientEventsService {
+  rpc LogEvents(LogEventsRequest) returns (LogEventsResponse);
+}
+
+enum Platform {
+    PLATFORM_UNSPECIFIED = 0;
+    PLATFORM_LINUX = 1;
+    PLATFORM_WINDOWS = 2;
+    PLATFORM_DARWIN = 3;
+}
+
+enum ClientEventType {
+    CLIENT_EVENT_TYPE_UNSPECIFIED = 0;
+    CLIENT_EVENT_TYPE_CLONE = 1;
+    CLIENT_EVENT_TYPE_PUSH = 2;
+    CLIENT_EVENT_TYPE_PULL = 3;
+}
+
+enum MetricID {
+    METRIC_ID_UNSPECIFIED = 0;
+    METRIC_ID_BYTES_DOWNLOADED = 1;
+    METRIC_ID_DOWNLOAD_MS_ELAPSED = 2;
+}
+
+enum AttributeID {
+    ATTRIBUTE_ID_UNSPECIFIED = 0;
+    ATTRIBUTE_ID_REMOTEURL = 1;
+}
+
+message ClientEventAttribute {
+    AttributeID id = 1;
+    string value = 2;
+}
+
+message ClientEventMetric {
+    oneof metric_oneof {
+        google.protobuf.Duration duration = 1;
+        int32 count = 2;
+    }
+
+    MetricID metric_id = 100;
```

Comment:
Why the jump in tag number?

Reply:
I was expecting what's inside the oneof to grow and I prefer to keep those consecutive... but really doesn't matter.